controllers/machineselector_controller.go

```diff
@@ -594,25 +596,34 @@ func (r *MachineInventorySelectorReconciler) setInvetorySelectorAddresses(ctx co
 	return nil
 }
 
-func (r *MachineInventorySelectorReconciler) ignoreIncrementalStatusUpdate() predicate.Funcs {
+func filterSelectorUpdateEvents() predicate.Funcs {
```

Comment:
These predicates are added to all watched resources, hence this applies for MachineInventories and MachineInventorySelectors